### PR TITLE
LPS-158376 [Raylife AP] Add tooltip and "All Changes Saved" status

### PR DESCRIPTION
**Pull request by joanagrego**
Related with -> [LPS-158376](https://issues.liferay.com/browse/LPS-158376)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfo.tsx [hunk 1]
```diff
@@ -46,23 +46,23 @@ const ContactInfo = () => {
 	const stateOptions = [
 		{
 			label: '',
-			value: '1',
+			value: '',
 		},
 		{
 			label: 'CHOOSE AN OPTION',
-			value: '2',
+			value: 'CHOOSE AN OPTION',
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfo.tsx:53]: Here you should use Camel Case to standardize. Example: **chooseAnOption**.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfo.tsx [hunk 2]
```diff
@@ -635,12 +664,13 @@ const ContactInfo = () => {
 					inline
 					onChange={(event: any) => {
 						const ownershipValue = event.target.value;
-						handleChangeField(form.ownership, ownershipValue);
+
+						handleChangeField('ownership', ownershipValue);
 					}}
 				>
-					<ClayRadio label="Rent" value="One" />
+					<ClayRadio label="Rent" value="Rent" />
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfo.tsx:671]: The values ​​must be written in all lowercase letters.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfo.tsx [hunk 3]
```diff
@@ -635,12 +664,13 @@ const ContactInfo = () => {
...
 
-					<ClayRadio label="Own" value="Two" />
+					<ClayRadio label="Own" value="Own" />
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfo.tsx:673]: The values ​​must be written in all lowercase letters.

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @joanagrego

**Review by silasbsi — APPROVED** [review]
LGTM 🔥 Thanks @joanagrego

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**